crypto/source_address_token.go

```diff
@@ -58,7 +58,11 @@ const stkKeySize = 16
 const stkNonceSize = 16
 
 // NewStkSource creates a source for source address tokens
-func NewStkSource(secret []byte) (StkSource, error) {
+func NewStkSource() (StkSource, error) {
```

Comment:
We agreed to keep them separate OOB, especially sind the server configs are not going to stay when we switch to TLS 1.3.